sysutils/libvirt/Portfile

```diff
@@ -17,56 +16,86 @@ long_description    A toolkit to interact with the virtualization \
 
 homepage            https://libvirt.org
 master_sites        ${homepage}/sources/
+use_xz yes
```

Comment:
Can we move this `yes` over to match the indentation of the other lines?

Reply:
Done.